composer.json

```diff
@@ -27,7 +27,6 @@
         "symfony/http-client": "5.4.*",
         "symfony/intl": "5.4.*",
         "symfony/mailer": "5.4.*",
-        "symfony/messenger": "5.4.*",
```

Comment:
This dependency added / required a new DB table, and since we're not using it at the moment, it's better to just remove it.